src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/Models/PSCognitiveServicesAccount.cs

```diff
@@ -55,7 +55,7 @@ public PSCognitiveServicesAccount(CognitiveServicesModels.CognitiveServicesAccou
 
         public string Etag { get; private set; }
 
-        public ProvisioningState? ProvisioningState { get; private set; }
+        public string ProvisioningState { get; private set; }
```

Comment:
This is the only meaningful change introduced by new SDK (3.0.0.0) but I don't think it is breaking.

Reply:
@felixwa This is a breaking change, right.  As in, I could have had scripts with things like:

```PowerShell
$account.ProvisioningState == [ProvisioningState]::Running
```

and such a script will now be broken.  I think you need to adapt this type in PSCognitiveServicesAccount so that this is not broken.  It is OK to add an additional string field and deprecate this field, but you will need to do some translation between the string field from the wire and this enumeration until the next breaking change release (build)

Reply:
@markcowl , the meaning of this PR is to introduce this change as we found we cannot enum all possible ProvisioningState so we changed the Swagger, SDK then this PS, change its type from an enum to a string.

I don't want to add back that enum in PS as it has gone in SDK also I cannot predicate possible values of that enum. If RP returns back a value not in the enum, PS throws an exception at client side.

Is there an option we wait "until the next breaking change release (build)"? If so we could wait but I really don't want to add that enum back in PS and only in PS.